Runtime/Plugins/webgl/Interface.jslib

```diff
@@ -19,11 +19,12 @@ mergeInto(LibraryManager.library, {
         if (targetDomObject != null) {
             targetDomObject.style.display = UTF8ToString(display);
             
-            	targetDomObject.style.left=left+(x*canvasWidth)+"px";
-		targetDomObject.style.top=bottom-((y+height)*canvasHeight)+"px";
-		targetDomObject.style.width = (width)*canvasWidth + "px";
+            targetDomObject.style.left=left+(x*canvasWidth)+"px";
+		    targetDomObject.style.top=bottom-((y+height)*canvasHeight)+"px";
```

Comment:
Stijl nitpick: waarom een andere indentatie?

Reply:
tabbing was verkeerd in zelfde block; meer indentatie gefixed zeg maar :P